packages/airnode-adapter/src/response-processing/extraction.test.ts

```diff
@@ -295,14 +295,14 @@ describe('respects maximum encoded size', () => {
   });
 
   describe('encoding int256 arrays', () => {
-    const maxInt256ArrayLength = 30;
+    const maxInt256ArrayLength = 510;
 
     it('is under the limit', () => {
```

Comment:
Test case names are switched. It should say `exceeds the limit` and `exceeds the limit` one should say `is under the limit`.

Reply:
Same goes for the string tests above.